files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js

```diff
@@ -24,12 +24,15 @@ const Settings = imports.ui.settings;
 const Pango = imports.gi.Pango;
 const AccountsService = imports.gi.AccountsService;
```

Comment:
```suggestion
```

Since you are cleaning code, you could remove this import since it isn't used anymore.